pkg/cmd/util/output/backup_describer.go

```diff
@@ -822,7 +826,11 @@ func groupByPhase(backups []velerov1api.PodVolumeBackup) map[string][]velerov1ap
 	phaseToGroup := map[velerov1api.PodVolumeBackupPhase]string{
 		velerov1api.PodVolumeBackupPhaseCompleted:  string(velerov1api.PodVolumeBackupPhaseCompleted),
 		velerov1api.PodVolumeBackupPhaseFailed:     string(velerov1api.PodVolumeBackupPhaseFailed),
+		velerov1api.PodVolumeBackupPhaseCanceled:   string(velerov1api.PodVolumeBackupPhaseCanceled),
 		velerov1api.PodVolumeBackupPhaseInProgress: "In Progress",
```

Comment:
Not sure about this line. Should we change it to `velerov1api.PodVolumeBackupPhaseInProgress: string(velerov1api.PodVolumeBackupPhaseInProgress),`?

Reply:
This one is not changed in this PR. I think it purposeful to make like this because it want to show it more friendly as "In Progress" instead of "InProgress"